hazelcast-hibernate/hazelcast-hibernate4/src/main/resources/META-INF/services/com.hazelcast.NodeInitializer

```diff
@@ -0,0 +1 @@
+com.hazelcast.instance.ExtendedNodeInitializer
```

Comment:
@bturner,

Since v3.4 (https://github.com/hazelcast/hazelcast/commit/7f826862ecc8e8308eeaf928dd5499595f2a4cd0) `WanReplicationService` creates/starts its threads on-demand. By default, when not used/touched, wan threads won't be created.  

Btw, `WanReplicationService` is not a completely enterprise feature, current open-source versions will remain as it is. Just new/enhanced version is enterprise.